pom.xml

```diff
@@ -0,0 +1,479 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.exasol</groupId>
+    <artifactId>kafka-connector-extension</artifactId>
+    <version>1.6.0</version>
+    <name>Exasol Kafka Connector Extension</name>
+    <description>Exasol Kafka Extension for accessing Apache Kafka</description>
+    <url>https://github.com/exasol/kafka-connector-extension/</url>
+    <parent>
+        <artifactId>kafka-connector-extension-generated-parent</artifactId>
+        <groupId>com.exasol</groupId>
+        <version>1.6.0</version>
+        <relativePath>pk_generated_parent.pom</relativePath>
+    </parent>
+    <properties>
+        <!-- Update this once the Avro4s version is updated -->
+        <!-- com.sksamuel.avro4s:avro4s-core_2.13:4.1.1 requires scala version: 2.13.3 -->
+        <scala.version>2.13.3</scala.version>
+        <scala.compat.version>2.13</scala.compat.version>
+        <sonar.sources>.</sonar.sources>
+        <sonar.inclusions>src/main/**</sonar.inclusions>
+    </properties>
+    <repositories>
+        <repository>
+            <id>io.confluent</id>
+            <url>https://packages.confluent.io/maven/</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
+    <dependencies>
+        <dependency>
+            <groupId>org.scala-lang</groupId>
+            <artifactId>scala-library</artifactId>
+            <version>${scala.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.exasol</groupId>
+            <artifactId>import-export-udf-common-scala_${scala.compat.version}</artifactId>
+            <version>1.1.1</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-simple</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.avro</groupId>
+                    <artifactId>avro</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-compress</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Are these exclusions common to all projects using `import-export-udf-common-scala_*`?
If yes: does it make sense to add the exclusions to the upstream project?
At least `slf4j-simple` can be excluded without consequences.

Reply:
That is good idea!
We should at least set the avro scope to provided. And also remove others since they are not causing any issues.

Created this ticket: https://github.com/exasol/import-export-udf-common-scala/issues/50